### PR TITLE
@ #53 | should display friendly log messages for the not configured v…

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #53 
please check this PR! Thank you

### CODE REVIEW

File: lib/teracy-dev-core/processors/variables.rb [hunk 1]
```diff
@@ -35,7 +35,12 @@ def process(settings)
 
         # see settings as a `sprintf-like formatting`
         # then inject values it and convert it back to YAML format
-        settings = eval(settings.to_s % formatedValues)
+
+        begin
+          settings = eval(settings.to_s % formatedValues)
+        rescue Exception => e
+          @logger.error(e)
```
Comment (hoatle) [inline lib/teracy-dev-core/processors/variables.rb]:
```ruby
@logger.error("#{e}, make sure it is configured in the `variables` settings")
abort
```

**Review by hoatle — CHANGES_REQUESTED** [review]
@hieptranquoc pls check my comments.

File: lib/teracy-dev-core/processors/variables.rb (hunk 1 above)
Comment (hieptranquoc) [reply lib/teracy-dev-core/processors/variables.rb]: I've updated! Please check again.
example error will show:
```
[TeracyDevCore::Processors::Variables][ERROR]: key{node_name_prefix} not found, make sure it is configured in the `variables` settings
```

**Comment by hieptranquoc** [comment]
@hoatle I've updated! Please check again! Thank you

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**